app/components/Activity/TransactionModal/TransactionModal.js

```diff
@@ -72,38 +72,53 @@ export default class TransactionModal extends React.PureComponent {
             }
           />
 
-          <Bar />
+          {item.num_confirmations > 0 && (
+            <>
+              <Bar />
 
-          <DataRow
-            left={<FormattedMessage {...messages.date_confirmed} />}
-            right={
-              <>
-                <FormattedDate
-                  value={item.time_stamp * 1000}
-                  year="numeric"
-                  month="long"
-                  day="2-digit"
-                />
-                <br />
-                <FormattedTime value={item.time_stamp * 1000} />
-              </>
-            }
-          />
+              <DataRow
+                left={<FormattedMessage {...messages.date_confirmed} />}
+                right={
+                  item.num_confirmations ? (
+                    <>
+                      <FormattedDate
+                        value={item.time_stamp * 1000}
+                        year="numeric"
+                        month="long"
+                        day="2-digit"
+                      />
+                      <br />
+                      <FormattedTime value={item.time_stamp * 1000} />
+                    </>
+                  ) : (
+                    <FormattedMessage {...messages.unconfirmed} />
+                  )
+                }
+              />
 
-          <Bar />
+              <Bar />
 
-          <DataRow
-            left={<FormattedMessage {...messages.address} />}
-            right={
-              <Link
-                className="hint--bottom-left"
-                data-hint={item.dest_addresses[0]}
-                onClick={() => this.showAddress(item.dest_addresses[0])}
-              >
-                <Truncate text={item.dest_addresses[0]} />
-              </Link>
-            }
-          />
+              <DataRow
+                left={<FormattedMessage {...messages.num_confirmations} />}
+                right={<FormattedNumber value={item.num_confirmations} />}
+              />
+
+              <Bar />
+
+              <DataRow
+                left={<FormattedMessage {...messages.address} />}
+                right={
+                  <Link
+                    className="hint--bottom-left"
+                    data-hint={item.dest_addresses[0]}
```

Comment:
Consider extracting `item.dest_addresses[0]` into a const to improve readability

Reply:
Updated